### ISSUE TITLE
Training problems about focal loss

### ISSUE THREAD

**Issue by blackantywj** [AlexeyAB/darknet#4884]
I use the binary cross entropy loss as a pre-training model to continue my training on focal loss. Is that appropriate? 
I used focal loss but found that the AP was not improved. May I ask how long it took everyone to achieve good results?
